### ISSUE TITLE
Put HorizontalRule component into Psammead

### ISSUE THREAD

**Issue by ChrisBAshton** [BBC-archive/psammead#53]
**Is your feature request related to a problem? Please describe.**
Blocked on [#1033](https://github.com/BBC-News/simorgh/pull/1033) - HorizontalRule component uses Text component in its Storybook render. The linked PR changes usage from Text to Paragraph.

Related to https://github.com/BBC-News/simorgh/issues/852 - one needs to happen before the other - doesn't matter which.

Corresponding 'pull from Psammead' issue in Simorgh: https://github.com/BBC-News/simorgh/issues/998.

**Describe the solution you'd like**
A PR in Psammead, which adds HorizontalRule component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests (if any) will be transferred from Simorgh.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

**Comment by sareh** [comment]
This can be picked up, since it shouldn't need to wait for the other issues to be completed.

**Comment by ChrisBAshton** [comment]
Superseded by https://github.com/BBC-News/simorgh/issues/1061.
